views/AddInventory.html

```diff
@@ -30,8 +35,11 @@
 
     <div class="row form-group">
         <div class="col-sm-4">
-            <label for="location_input">Location</label>
-            <input id="location_input" class="form-control" title="The location this inventory will be stored at."></input>
+            <form>
+                        Location
+                        <input type="text" id="location_input" name="location_input" value="A-1"
```

Comment:
can we use a different kind of dropdown or change the icon somehow? it doesn't seem to display properly.

![image](https://cloud.githubusercontent.com/assets/8051470/9101389/16e0f4e2-3b9b-11e5-8bbc-a77177e9d6b6.png)